src/TodoList.js

```diff
@@ -17,17 +18,17 @@ const todoList = [
 
 function TodoList() {
   return (
+    <> 
```

Comment:
Nitpick (doesn't require change):
You can remove this fragment (```<>```) since the `ul` tag wraps all children elements. If you have an element outside of the `ul` element, then you'd need a fragment because all elements must have a wrapping parent element.

Here's an example, where a fragment is needed:
```jsx
<>
  <ul>
    {todoList.map(todo => (
      <TodoListItem
        key={todo.id}
        todo={todo}
      /> 
    ))}
  </ul>
  <p>
    This element isn't contained within the ul element and therefore, both the ul and p tags require a wrapping parent element
  </p>
</>
```